.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
What is this for?

Reply:
Oh, my editor, when I edit `index.html`, keeps a backup as `index.html~`
It's fairly normal to have a more detailed `.gitignore` with a bunch of things like `*.bak`, `*.o`, `OLD/*`, `*.cert`, `*.tmp` or other stuff that you **DON'T** want to include in `git add` / `git commit` / etc. Most git projects I work on have a **much** more detailed `.gitignore` but I thought it was important to at least get one that can ignore my own editor files. Feel free to send a PR for more if your editor uses something else, or if there's any other rogue files you think we should ignore going forward.

Reply:
I'll admit it's not relevant to #181 and should arguably have been in a different PR, but... Meh!   :man_shrugging: